app/views/pages/index.jade

```diff
@@ -101,14 +101,24 @@ script#attribute-template(type='text/template')
     <!-- /ko -->
   <!-- /ko -->
 script#switch-attribute-template(type='text/template')
+  <!-- ko if: !$data.hidden -->
   <!-- ko if: $data.type !== 'boolean' -->
   span.attribute(data-bind="tooltip: $data.tooltipHtml, css: {outOfDate: $data.outOfDate()}")
+    <!-- ko if: $data.shouldDisplayAcronym() -->
+    span.acronym(data-bind="text: $data.displayAcronym()")
+    <!-- /ko -->
+    <!-- ko if: $data.shouldDisplayIcon() -->
+    span.attribute-icon(data-bind="css: $data.getIconClass()")
+    <!-- /ko -->
+    <!-- ko if: $data.shouldDisplayValue() -->
     span.val(data-bind="text: $data.displayValueText()")
     span.unit(data-bind="text: $data.displayUnitText()")
+    <!-- /ko -->
     <!-- ko if: $data.shouldDisplaySparkline()  -->
     span.sparkline(data-bind="sparkline: {data: $data.sparklineHistory, tooltipFormatter: $data.tooltipFormatter}")
     <!-- /ko -->
   <!-- /ko -->
+  <!-- /ko -->
```

Comment:
It looks like there is no real difference between the attribute-template and the switch-attribute-template. I think it would be better to use the attribute-template for the switch, too and move the if statement to the switch template. I agree with changing the if condition to check the attribute name instead of type.